gan_utils.py

```diff
@@ -52,13 +54,12 @@ async def split_animation(url, output, fps=30):
 
 def get_vocab(txt_dir_path, filter_tags=None, top=None, splitter=", "):
     # Also supports .vocab file:
-    if os.path.isfile(txt_dir_path):
+    if Path(txt_dir_path).is_file():
```

Comment:
what is the benefit of using pathlib here over standard os?

Reply:
It's a more moderm implementation of the same thing, so it's easier to manage paths, in this usage there's no much advantage at all